app/helpers/PermissionHints.php

```diff
@@ -24,8 +25,8 @@ public static function generate($aclModule, $subject)
     {
         foreach (static::getAclMethods($aclModule) as $method) {
             $parameter = $method->getParameters()[0];
-            /** @var ?ClassType $classObj */
-            $classObj = $parameter->getClass();
```

Comment:
`getClass` is deprecated in PHP 8, I am leaving this as a first early bird here...